commands/narrator/srole.js

```diff
@@ -207,11 +207,10 @@ module.exports = {
         shuffle(finalRoleList)
         let sorcChats = []
         for (let k = 0; k < alive.members.size; k++) {
-            let theirRole = finalRoleList[k]
+            let theirRole = finalRoleList[k].name
```

Comment:
theirRole is always a string iirc, is there a spot that runs getRole later that we can use the name from?